client/src/components/pages/MembersPage/components/CreateForm/CreateFormStyledComponents.js

```diff
@@ -0,0 +1,154 @@
+import styled from 'styled-components'
+
```

Comment:
There is some repetition in this styling, maybe you can reuse some common behaviour?